catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/ui/activity/utils/FormulaEditorCategoryListDataInteractionWrapper.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Catroid: An on-device visual programming system for Android devices
+ * Copyright (C) 2010-2017 The Catrobat Team
+ * (<http://developer.catrobat.org/credits>)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * An additional term exception under section 7 of the GNU Affero
+ * General Public License, version 3, is available at
+ * http://developer.catrobat.org/license_additional_term
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.catrobat.catroid.uiespresso.ui.activity.utils;
+
+import android.support.test.espresso.DataInteraction;
+
+import org.catrobat.catroid.uiespresso.util.wrappers.DataInteractionWrapper;
+
+import static android.support.test.espresso.Espresso.onData;
+
+import static org.catrobat.catroid.uiespresso.util.matchers.FormulaEditorCategoryListMatchers.isFormulaEditorCategoryListView;
+import static org.catrobat.catroid.uiespresso.util.matchers.FormulaEditorCategoryListMatchers.isFormulaEditorListItem;
+import static org.hamcrest.Matchers.instanceOf;
+
+public class FormulaEditorCategoryListDataInteractionWrapper extends DataInteractionWrapper {
+
+	public FormulaEditorCategoryListDataInteractionWrapper(DataInteraction dataInteraction) {
+		super(dataInteraction);
+	}
+
+	public static DataInteraction onFormulaEditorCategory() {
```

Comment:
bump.
i did not meant just renaming this, but leaving it extending the DataInteractionWrapper.

please take
FormulaEditorCategoryListDataInteractionUtils.java
ProjectListDataInteractionUtils.java
ScriptListDataInteractionUtils.java

and move the one contained functions somewhere meaningful, in this case, since each of them is only used in single test classes, put them in the testclasses.

so in general, if a utility function is used in - 
- one test class, put it there
- multiple test classes in the same package, make a utility function class in this package, make it package private
- multiple test classes spanning multiple distinct packages, make a global utility file with a meaningful name in the utils packag.